.azure-pipelines/templates/set-up-integrations.yml

```diff
@@ -7,3 +7,4 @@ steps:
   displayName: 'Set up integration requirements'
   env:
     DOCKER_PASSWORD: $(DOCKER_PASSWORD)
+    ORACLE_DOCKER_PASSWORD: $(ORACLE_DOCKER_PASSWORD)
```

Comment:
Do we need a specific password for oracle?

Reply:
Yes, it's also a different registry. https://github.com/DataDog/integrations-core/pull/5179/files#diff-47b5f66eed96676d1b65faabda07c99eR6